geode-core/src/test/java/org/apache/geode/distributed/internal/LocatorLoadSnapshotJUnitTest.java

```diff
@@ -631,28 +633,93 @@ public void testFindBestServersCalledWithNegativeCount() {
   }
 
   @Test
```

Comment:
Hi @pivotal-jbarrett , thanks for the review. Upgrading this test to JUnit 5 would be tricky because it uses @Rule annotation, which is replaced by @ExtendWith annotation. It would be necessary to implement the new interfaces (AfterEachCallback and BeforeEachCallback) to GfshCommandRule and ClusterStartupRule classes. I think that this should be a part of a separate ticket since this is not just a minor adjustment. What do you think?